python-test/features/steps/ui_utils.py

```diff
@@ -0,0 +1,42 @@
+from hamcrest import *
+from selenium import webdriver
+
+
+def go_to_page(page, context, ignore_ssl_and_certificate_errors=True):
```

Comment:
I understand that `ignore_ssl_and_certificate_errors` is a parameter, however by default we're ignoring possible SSL errors. But we should always verify the SSL certificate, and only in the specific case of using self-signed certificates we should ignore it.
This means we should check SSL certificates by default and only ignore errors if we're testing localhost.
If you'd like we can discuss the best approach to this.

Reply:
Thanks for the tips. I think it's better this way.